cdc/processor/manager.go

```diff
@@ -129,12 +134,20 @@ func (m *Manager) Tick(stdCtx context.Context, state orchestrator.ReactorState)
 
 func (m *Manager) closeProcessor(changefeedID model.ChangeFeedID) {
 	if processor, exist := m.processors[changefeedID]; exist {
+		startTime := time.Now()
+		captureID := processor.captureInfo.ID
 		err := processor.Close()
 		if err != nil {
 			log.Warn("failed to close processor",
 				zap.String("changefeed", changefeedID),
 				zap.Error(err))
 		}
+		costTime := time.Since(startTime)
```

Comment:
May be it is better to move the warn log above the err handle logic.

Reply:
done